1-Authentication/2-sign-in-b2c/server.js

```diff
@@ -2,11 +2,31 @@ const express = require('express');
 const morgan = require('morgan');
 const path = require('path');
 
+const rateLimit = require('express-rate-limit');
+
 const DEFAULT_PORT = process.env.PORT || 6420;
 
 // initialize express.
 const app = express();
 
+/**
+ * HTTP request handlers should not perform expensive operations such as accessing the file system,
+ * executing an operating system command or interacting with a database without limiting the rate at
+ * which requests are accepted. Otherwise, the application becomes vulnerable to denial-of-service attacks
+ * where an attacker can cause the application to crash or become unresponsive by issuing a large number of
+ * requests at the same time. For more information, visit: https://cheatsheetseries.owasp.org/cheatsheets/Denial_of_Service_Cheat_Sheet.html
+ */
+const limiter = rateLimit({
```

Comment:
This is not a web API sample, and the server here is for demonstration only -we shouldn't bother with rate-limiting